app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -32,6 +32,18 @@ def create
     )
     if @planning_application.valid? && @planning_application.save!
       upload_documents(params[:files])
+
+      api_user = User.find_or_create_by!(
```

Comment:
We don't want to be creating API users here! `current_api_user` is set in `Api::V1::ApplicationController`